<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UISettingsGUI</class>
 <widget class="QWidget" name="UISettingsGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Basic</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Language file</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QLineEdit" name="lineEdit_LANGFILE"/>
          </item>
          <item row="0" column="1">
           <widget class="QToolButton" name="toolButton_LANGBROWSE">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QComboBox" name="comboBox_LANGS"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Theme</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QComboBox" name="comboBox_THEMES"/>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButton_TEST">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Test</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Application font</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QLineEdit" name="lineEdit_APPFONT"/>
          </item>
          <item row="0" column="1">
           <widget class="QToolButton" name="toolButton_APPFONTBROWSE">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Chat</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="4">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Text color</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QListWidget" name="listWidget_CHATCOLOR"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QCheckBox" name="checkBox_CHATJOINS">
         <property name="text">
          <string>Show joins</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="4">
        <widget class="QCheckBox" name="checkBox_CHATHIDDEN">
         <property name="text">
          <string>Show hidden users</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Maximum paragraphs</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="3">
        <widget class="QSpinBox" name="spinBox_PARAGRAPHS">
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="4">
        <widget class="QCheckBox" name="checkBox_IGNOREPMHUB">
         <property name="text">
          <string>Ignore private messages from Hub</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="4">
        <widget class="QCheckBox" name="checkBox_IGNOREPMBOT">
         <property name="text">
          <string>Ignore private messages from Bot</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="4">
        <widget class="QCheckBox" name="checkBox_REDIRECTPMBOT">
         <property name="text">
          <string>Redirect messages from Bot to the main chat</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="4">
        <widget class="QCheckBox" name="checkBox_KEEPFOCUS">
         <property name="text">
          <string>Do not change focus on new private message</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
